style/style.css

```diff
@@ -77,3 +62,81 @@
   padding: 0 10px; /* pushes the text in from the left and right */
   font-size: 0.9rem;
 }
+
+@media only screen and (min-width: 720px) and (max-width: 1023px) {
```

Comment:
Great work, I noticed that when I zoomed out to 75% the cards was in the right size however when it's 100% the text gets out of the card